install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -75,7 +75,7 @@ spec:
 {{- end }}
         - name: istio-proxy
           image: "{{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}"
-          imagePullPolicy: {{ .Values.global.proxy.imagePullPolicy }}
+          imagePullPolicy: {{ .Values.global.imagePullPolicy }}
```

Comment:
I prefer that you add the `imagePullPolicy` for all of the deployments in istio charts.

Reply:
@gyliu513 From what I see `imagePullPolicy` is already in the deployments in Istio charts. Here just a wrong path was used: instead of `.Values.global.proxy.imagePullPolicy` it should be `.Values.global.imagePullPolicy`.

Reply:
seems we want to use one parameter to control all image pull policy for control panel, ok to me.